client/config/environment.js

```diff
@@ -37,17 +38,15 @@ module.exports = function(environment) {
     ENV.contentSecurityPolicy = {
       'default-src': "'none'",
       // add the domain here like analytics.google.com or data: or 'self'
-      'script-src':  "'self' 'unsafe-eval' 'https://www.google.com' ''",
+      'script-src':  "'self' 'unsafe-eval' 'unsafe-inline' www.google.com www.google-analytics.com",
```

Comment:
I'm not sure this is a good idea from a security perspective. It basically opens us up to XSS attacks.  Check out "Inline Code Considered Harmful"  in http://www.html5rocks.com/en/tutorials/security/content-security-policy/